packages/lib-classifier/src/components/Classifier/components/Banners/components/SubjectSetProgressBanner/SubjectSetProgressBanner.stories.js

```diff
@@ -1,79 +1,171 @@
+      themeMode={(dark) ? 'dark' : 'light'}
+    >
+      <Provider classifierStore={store}>
+        <Box background={{ dark: 'dark-3', light: 'light-3' }} width='large'>
+          <Banners />
+          <img src="https://placekitten.com/800/400" alt='placeholder' />
+        </Box>
+      </Provider>
+    </Grommet>
+  )
+}
+
+Default.args = {
+  already_seen: false,
+  dark: false,
+  retired: false
+}
+
+export function WithVisiblePriorityMetadata({ already_seen, dark, retired }) {
+  const priority = { priority: 37 }
+  const store = buildMocks({ already_seen, priority, retired })
+  return (
+    <Grommet
+      theme={Object.assign({}, zooTheme, { dark })}
+      themeMode={(dark) ? 'dark' : 'light'}
+    >
+      <Provider classifierStore={store}>
+        <Box background={{ dark: 'dark-3', light: 'light-3' }} width='large'>
+          <Banners />
+          <img src="https://placekitten.com/800/400" alt='placeholder' />
+        </Box>
+      </Provider>
+    </Grommet>
+  )
+}
+
+export function WithVisiblePriorityMetadataAndRetired({ already_seen, dark, retired }) {
+  const priority = { priority: 37 }
+  const store = buildMocks({ already_seen, priority, retired })
+  return (
+    <Grommet
+      theme={Object.assign({}, zooTheme, { dark })}
+      themeMode={(dark) ? 'dark' : 'light'}
+    >
+      <Provider classifierStore={store}>
+        <Box background={{ dark: 'dark-3', light: 'light-3' }} width='large'>
+          <Banners />
+          <img src="https://placekitten.com/800/400" alt='placeholder' />
+        </Box>
+      </Provider>
+    </Grommet>
+  )
+}
+
+WithVisiblePriorityMetadataAndRetired.args = {
+  already_seen: false,
+  dark: false,
+  retired: true
+}
+
+export function WithVisiblePriorityMetadataAndAlreadySeen({ already_seen, dark, retired }) {
+  const priority = { priority: 37 }
+  const store = buildMocks({ already_seen, priority, retired })
+  return (
+    <Grommet
+      theme={Object.assign({}, zooTheme, { dark })}
+      themeMode={(dark) ? 'dark' : 'light'}
+    >
+      <Provider classifierStore={store}>
+        <Box background={{ dark: 'dark-3', light: 'light-3' }} width='large'>
+          <Banners />
+          <img src="https://placekitten.com/800/400" alt='placeholder' />
+        </Box>
+      </Provider>
+    </Grommet>
+  )
+}
+
+WithVisiblePriorityMetadataAndAlreadySeen.args = {
+  already_seen: true,
+  dark: false,
+  retired: false
+}
+
+export function WithRetiredSubject({ already_seen, dark, retired }) {
+  const store = buildMocks({ already_seen, retired })
+  return (
+    <Grommet
+      theme={Object.assign({}, zooTheme, { dark })}
+      themeMode={(dark) ? 'dark' : 'light'}
+    >
+      <Provider classifierStore={store}>
+        <Box background={{ dark: 'dark-3', light: 'light-3' }} width='large'>
+          <Banners />
+          <img src="https://placekitten.com/800/400" alt='placeholder' />
+        </Box>
+      </Provider>
+    </Grommet>
+  )
+}
```

Comment:
Does this duplicate the standalone story for the retired subject banner?

I've been wondering, after working on the other PRs for banners, if we should move the tests and stories up one level, and just have a set of tests and stories which render the `Banners` component for different combinations of subject and workflow, like @srallen has done here.